packages/modals/src/styled/StyledModal.ts

```diff
@@ -47,9 +47,8 @@ const sizeStyles = (props: IStyledModalProps & ThemeProps<DefaultTheme>) => {
   const largeWidth = 800;
 
   return css`
-    width: ${props.isLarge ? `${largeWidth}px` : `${defaultWidth}px`};
-    @media (max-width: ${props.isLarge ? `${largeWidth - 1}px` : props.theme.breakpoints.md}) {
-      ${props.theme.rtl ? 'right' : 'left'}: ${props.theme.space.base * 6}px;
+    @media (min-width: ${props.isLarge ? `${largeWidth - 1}px` : props.theme.breakpoints.sm}) {
+      width: ${props.isLarge ? `${largeWidth}px` : `${defaultWidth}px`};
```

Comment:
Was an attempt made to use the [`mediaQuery`](https://garden.zendesk.com/components/utilities#mediaquery) utility here? In general, that would be preferable.

Reply:
No, but I'm looking into it now. It seems like `mediaQuery` would not support a custom breakpoint (e.g. `largeWidth - 1px` or `799px`).

I don't recall why `799px` was used for the `isLarge` breakpoint. If one of the theme breakpoints can be used instead, then media could look like: `mediaQuery('up', props.isLarge ? 'md' : 'sm', props.theme)`.

Reply:
Might require a separate PR to add some flexibility to the `mediaQuery` function.

Reply:
@james-sann mind if I loop you into this for the design decision?

The current media query for the `isLarge` modal is custom and targets `@media (min-width: 799px)`. `799px` is a custom computed value. In order to use the `mediaQuery` utility suggested in this thread, the media query size must be one from Garden’s [theme object](https://garden.zendesk.com/components/theme-object#default_theme) and not custom. 

@james-sann, could you help us figure out if Garden can use `theme.breakpoints.md` instead of a custom `799px`? From the engineering perspective, I think using `theme.breakpoints.md` (aka `768px`) would be just fine. But I wanted to run it by design first since this would change the media query target of what is currently in production.